File: countRotations.java
import java.util.*;
public class Main 
{
    public static void main(String[] args)
    {
        Scanner sc=new Scanner(System.in);
        int indexI,minElement,lengthOfArray,indexOfMinElement;
        System.out.println("Enter length of array");
        lengthOfArray=sc.nextInt();
        int array[]=new int[lengthOfArray];
        for(indexI=0;indexI<lengthOfArray;indexI++)
        array[indexI]=sc.nextInt();minElement=array[0];indexOfMinElement=0;
        for(indexI=1;indexI<lengthOfArray;indexI++)
        {
            if(array[indexI]<minElement)
            {
                minElement=array[indexI];
                indexOfMinElement=indexI;
            }
        }
        System.out.println(indexOfMinElement);
    }
}

/* 
Input/Output - 
Enter length of array                                                                                                                      
6                                                                                                                                          
5 6 1 2 3 4                                                                                                                                
2 */                                                                                                                                         
     

File: countRotations.py
n=int(input('Enter number of elements\n'))
a=[]
print('Enter n elements')
for i in range (0,n):
    a.append(int(input()))
minElement= a[0]
minELementIndex = 0
for i in range(1,n):
    if a[i]<minElement:
        minElement=a[i]
        minELementIndex = i
print(minELementIndex)